docs/plugins/mcp.mdx

```diff
@@ -31,6 +31,8 @@ This plugin adds [Model Context Protocol](https://modelcontextprotocol.io/docs/g
   - You can allow / disallow `find`, `create`, `update`, and `delete` operations for each collection
   - You can to allow / disallow capabilities in real time
   - You can define your own Prompts, Tools and Resources available over MCP
+- Full support for Payload's localization features with `locale` and `fallbackLocale` parameters
+- MCP server supporting HTTP (with REDIS and STDIO under consideration)
```

Comment:
We are only supporting `HTTP` right now.

REDIS specifically is not the only option for Server Side Events(SSE), its availability is an artifact of an underlying dependency we are using at the moment.

STDIO is on the roadmap for folks who are doing local AI dev.